tools/dump_tool/dump_tool.c

```diff
@@ -392,6 +439,25 @@ int main(int argc, char *argv[])
 			printf("\nFilesystem Dump received successfully\n");
 			fflush(stdout);
 		}
+
+		if (DUMP_FLAGS & EXTUSERFS_DUMP_FLAG) {
+			printf("\nDUMPING EXTERNAL USERFS DUMP PARTITION\n");
+			fflush(stdout);
+			/* Handshake for synchronization before External Userfs Dump */
```

Comment:
can you give a space below and up?
it will increase the readability for comment.

Reply:
Added suitable spaces